utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -16,8 +16,21 @@ annotation class SplashScreenWelcomeMsg
 val SPLASH_SCREEN_WELCOME_MSG = "splash_screen_welcome_msg"
 val SPLASH_SCREEN_WELCOME_MSG_DEFAULT_VALUE = false
 
+/**
+ * Qualifier for the platform parameter that defines the time period in hours, after which the
+ * [PlatformParameterSyncUpWorker] will run again.
+ */
 @Qualifier
 annotation class SyncUpWorkerTimePeriod
```

Comment:
If this is in hours, please include that in the name (e.g. ``SyncUpWorkerTimePeriodHours`` to make it clear at the injection site).

Reply:
Done

Reply:
Done